### ISSUE TITLE
Crone After Hour Is Not Working Properly

### ISSUE THREAD

**Issue by MalikSamiAwan** [agilord/cron#46]
cron.schedule(Schedule.parse('0 * * * *'), () async {
//my code
});

above code is always throwing exception

**Comment by isoos** [comment]
what's the exception?

**Comment by MalikSamiAwan** [comment]
@isoos 
closed

**Comment by bahman2000** [comment]
@MalikSamiAwan Yes, this format is correct for hourly job at minute 00. Which platform did you test?

**Comment by MalikSamiAwan** [comment]
@bahman2000 Thanks, i was testing it on MAC OS
